support/ci/specs.sh

```diff
@@ -75,8 +75,8 @@ find /hab/pkgs/hab_travis/hab/ -type f -name hab -exec cp {} ${TEST_BIN_DIR} \;
 find /hab/pkgs/hab_travis/hab-sup/ -type f -name hab-sup -exec cp {} ${TEST_BIN_DIR} \;
 
 
-# TODO
-# https://docs.travis-ci.com/user/pull-requests
+TODO
```

Comment:
I think these are syntax errors when not commented out. I also am not sure what the TODO was trying to tell us